dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7100,6 +7100,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey DORA_READ_VIRTUAL_BLOCK_SIZE =
+      dataSizeBuilder(Name.DORA_READ_VIRTUAL_BLOCK_SIZE)
+          .setDefaultValue("0MB")
```

Comment:
can you elaborate what will happen is this property is set to 0mb? does it mean this feature is disabled?